client/src/docs-ui/component-playground/component-playground.tsx

```diff
@@ -0,0 +1,21 @@
+import {Component, h, Prop} from "@stencil/core";
+import {AuthenticatorWithSlots} from "./custom-components/authenticator-with-slots";
+
+@Component({
+  tag: "docs-component-playground",
+})
+export class DocsComponentPlayground {
+  /** Name of component used in the playground */
+  @Prop() readonly componentName: string;
+
+  renderCustomComponent(componentName: string) {
```

Comment:
If there's a fixed set of possible strings, can you specify that instead of `string`. Let's narrow it... this requires changing the type on line 9 as well.

Reply:
Good call.